developer-hub/src/polkadot_sdk/mod.rs

```diff
@@ -0,0 +1,126 @@
+//! # Polkadot SDK
+//!
+//! The [Polkadot SDK repository](https://github.com/paritytech/polkadot-sdk) provides the main
+//! resources needed to start building on the [Polkadot network](https://polkadot.network), a
+//! multi-chain blockchain platform that enables different blockchains to interoperate and share
+//! information in a secure and scalable way.
+//!
+//! [![StackExchange](https://img.shields.io/badge/StackExchange-Polkadot%20and%20Substrate-222222?logo=stackexchange)](https://substrate.stackexchange.com/)
+//!
+//! [![awesomeDot](https://img.shields.io/badge/polkadot-awesome-e6007a?logo=polkadot)](https://github.com/Awsmdot/awesome-dot)
+//! [![awesomeDot](https://img.shields.io/badge/polkadot-wiki-e6007a?logo=polkadot)](https://wiki.polkadot.network/)
+//! [![awesomeDot](https://img.shields.io/badge/polkadot-forum-e6007a?logo=polkadot)](https://forum.polkadot.network/)
+//!
+//! [![RFCs](https://img.shields.io/badge/fellowship-RFCs-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/rfcs)
+//! [![Runtime](https://img.shields.io/badge/fellowship-runtimes-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/runtimes)
+//! [![Manifesto](https://img.shields.io/badge/fellowship-manifesto-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/manifesto)
+//!
+//! ## Getting Started
+//!
+//! The primary way to get started with the Polkadot SDK is to start writing a Substrate-based
+//! runtime using FRAME. See:
```

Comment:
Having some sort of representation of running nodes is a must as noted in https://github.com/paritytech/polkadot-sdk/pull/2102/files#r1381155156, but afterwards I am not sure. 

We need one good and simple and bulletproof means of running networks/binaries in this platform. It could be zombienet, CLI tests, or last resort just bash commands that we pray remain up to date (should really be avoided). Once that is done, we can think of where we go next.